### ISSUE TITLE
Support for multiple instances... almost?

### ISSUE THREAD

**Issue by johnray** [ngoduykhanh/wireguard-ui#311]
I have two wg interfaces running on a server - each tied to a different static public IP so that I can do port forwarding from each of the public IPs.  That's fine. 

I had hoped that even though wireguard-ui doesn't support multiple interfaces, that it would allow me to run two separate instances.  I can /sorta/ do this by using a different -base-path for the wireguard-ui db folder structure and a different server port for wireguard-ui.  Unfortunately, no matter what I do, I get a 404 error after setting it up.  I can see the db directory and files being created, but the second instance of wireguard-ui starts, runs, and just 404s on every request.  

I've tried world readable/writeable directories for the base-path, I've tried dumping it in /tmp... just 404.

I think I managed to make it work briefly by NOT using base-path and just starting it manually while in the directory where I wanted the db files written (seems to create the db files wherever you happen to be when running it manually).  At any rate, that (probably my fault) didn't quite work because there seemed to be some interactions between the original instance and the second instance...

TL;DR - base-path doesn't, IMO, seem to work properly for me.  One instance spins up just fine with the default paths, the second runs with a new base-path, but answers all requests with a 404.

**Comment by andycandy-de** [comment]
Is docker an option for you? I think it could be easier to create multiple instances with docker.
